deployment/E_heroku.md

```diff
@@ -1 +1,308 @@
+remote:
+remote: -----> Fetching Elixir v1.0.4
+remote: -----> Installing Elixir v1.0.4 (changed)
+remote: -----> Installing Hex
+remote: 2015-07-07 00:04:00 URL:https://s3.amazonaws.com/s3.hex.pm/installs/1.0.0/hex.ez [262010/262010] ->
+"/app/.mix/archives/hex.ez" [1]
+remote: * creating /app/.mix/archives/hex.ez
+remote: -----> Installing rebar
+remote: * creating /app/.mix/rebar
+remote: -----> Fetching app dependencies with mix
+remote: Running dependency resolution
+remote: Dependency resolution completed successfully
+remote: [...]
+remote: -----> Compiling
+remote: [...]
+remote: Generated phoenix_heroku app
+remote: [...]
+remote: Consolidated protocols written to _build/prod/consolidated
+remote: -----> Creating .profile.d with env vars
+remote: -----> Fetching custom git buildpack... done
+remote: -----> Phoenix app detected
+remote:
+remote: -----> Loading configuration and environment
+remote:        Loading config...
+remote:        WARNING: phoenix_static_buildpack.config wasn't found in the app
+remote:        Using default config from Phoenix static buildpack
+remote:        Will use the following versions:
+remote:        * Node 0.12.4
+remote:        Will export the following config vars:
+remote:        * Config vars DATABASE_URL
+remote:        * MIX_ENV=prod
+remote:
+remote: -----> Installing binaries
+remote:        Downloading node 0.12.4...
+remote:        Installing node 0.12.4...
+remote:        Using default npm version
+remote:
+remote: -----> Building dependencies
+remote:        [...]
+remote:        Running default compile
+remote:               Building Phoenix static assets
+remote:        07 Jul 00:06:22 - info: compiled 3 files into 2 files, copied 3 in 3616ms
+remote:        Check your digested files at 'priv/static'.
+remote:
+remote: -----> Finalizing build
+remote:        Creating runtime environment
+remote:
+remote: -----> Discovering process types
+remote:        Procfile declares types     -> (none)
+remote:        Default types for Multipack -> web
+remote:
+remote: -----> Compressing... done, 82.1MB
+remote: -----> Launching... done, v5
+remote:        https://mysterious-meadow-6277.herokuapp.com/ deployed to Heroku
+remote:
+remote: Verifying deploy... done.
+To https://git.heroku.com/mysterious-meadow-6277.git
+ * [new branch]      master -> master
+```
+
+Typing `heroku open` in the terminal should launch a browser with the Phoenix welcome page opened.
+
+And that's it!
+
+## Usefull Heroku Commands
+
+We can look at the logs of our application by running the following command in our project directory:
+
+```console
+$ heroku logs # use --tail if you want to tail them
+```
+
+We can also start an IEx session attached to our terminal for experimenting in our app's environment:
+
+```console
+$ heroku run iex -S mix
+```
+
+In fact, we can run anything using the `heroku run` command, like the Ecto migration task for instance:
+
+```console
+$ heroku run mix ecto.migrate
```

Comment:
Was thinking just a mention but avoid going in-depth. Just a thought. Anyway, thanks for this! :)